src/signals/incident/components/form/MapSelectors/Asset/Selector/WfsLayer/AssetLayer/AssetLayer.tsx

```diff
@@ -32,6 +32,7 @@ import { featureTolocation } from 'shared/services/map-location'
 import MarkerCluster from 'components/MarkerCluster'
 
 import configuration from 'shared/services/configuration/configuration'
+import { select as SelectIcon } from 'signals/incident/definitions/wizard-step-2-vulaan/verlichting-icons'
```

Comment:
Can we avoid having this dependency in the `AssetLayer` component? This component is independent and should stay independent. Adding `SelectIcon` reduces the generic nature of `AssetLayer`.

Reply:
this icon is a general selection icon. I will move it to a more general location